nook/functions/doc_translator/translation_engine.py

```
@@ -0,0 +1,312 @@
+                processed_text,
+                system_instruction=self.config.system_prompt,
+                temperature=self.config.temperature,
+                max_output_tokens=self.config.max_tokens
+            )
+
+            # Post-process to ensure consistency
+            final_text = self._postprocess_translation(text, translated)
+
+            return final_text
+
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)